code/modules/modular_computers/file_system/programs/ntmonitor.dm

```diff
@@ -1,6 +1,6 @@
 /datum/computer_file/program/ntnetmonitor
-	filename = "ntmonitor"
-	filedesc = "NTNet Diagnostics and Monitoring"
+	filename = "wirecarp"
+	filedesc = "WireCarp"
```

Comment:
Why not `NetCarp` ? Since `wire` gives some association with cable coil and machine wires.

Reply:
This one is a pun of the real life program WireShark.

Reply:
Ok